docs/reference/remap/functions/parse_timestamp.cue

```diff
@@ -30,9 +30,9 @@ remap: functions: parse_timestamp: {
 		{
 			title: "Parse timestamp"
 			source: #"""
-				parse_timestamp("10-Oct-2020 16:00", format: "%v %R")
+				parse_timestamp!("10-Oct-2020 16:00", format: "%v %R")
 				"""#
-			return: "2020-10-10T16:00:00Z"
+			return: "2020-10-10T14:00:00Z"
```

Comment:
This is going to fail on the CI (time zone differences).

The function currently sets the time zone to local, but perhaps we should set it to `UTC` until we have an option to override the time zone?